hack/clean_vendor.sh

```diff
@@ -1,4 +1,15 @@
-find vendor -type f -not -name "*.go" -not -name "LICENSE" -not -name "*.s" -not -name "PATENTS" -not -name "*.h" -not -name "*.c" | xargs -I {} rm {}
-# delete all test files
+# delete internal vendor folders
+find vendor -type d -name "_vendor" | xargs -I {} rm -r {}
+# delete all files that are not go, c, h, or legal
+find vendor -type f -not -name "*.go" -not -name "NOTICE*" -not -name "COPYING*"  -not -name "LICENSE*" -not -name "*.s" -not -name "PATENTS*" -not -name "*.h" -not -name "*.c" | xargs -I {} rm {}
+# delete all generated files
 find vendor -type f -name "*_generated.go" | xargs -I {} rm {}
```

Comment:
do we have the generated Go files?

Reply:
This was in the tidb project. My strategy was to keep this file the same for both. But we could remove the rules that have no effect.